src/components/profile/profileHeader.js

```diff
@@ -39,12 +52,17 @@ export default class ProfileHeader extends Component {
             </div>
             <span className="nav-block" />
           </div>
-          <Description
-            name={name}
-            state={state}
-            country={country}
-            profilelink={profilelink}
-          />
+
+          {firstName ? (
+            <Description
+              name={`${firstName} ${lastName}`}
+              state={state}
+              country={country}
+              profilelink={profilelink}
+              click={this.handleClick}
```

Comment:
When doing this, in updated versions of react, you don't have to add this to your constructor
`this.handleClick = this.handleClick.bind(this);`
Instead what you'd do is
```
handleClick = () => {
    this.setState(prevState => ({ show: !prevState.show }));
  }
```

Reply:
hmm ...ok.
are we using the updated version?

Reply:
but when be are calling it, would be ```click={this.handleClick}``` or click={handleClick}